cmd/file_lint.go

```diff
@@ -110,11 +113,16 @@ func executeLint(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to read input file '%s'; %w", cmdLintInputFilename, err)
 	}
 
+	functionsFlag, _ := cmd.Flags().GetString("functions")
```

Comment:
No need to create this `functionsFlag` variable. When defining the flag like this below:

`lintCmd.Flags().StringVarP(&cmdCustomFunctionsPath, ...`

you pass the address of the global variable `cmdCustomFunctionsPath`, so that is where the value of the flag will be stored. So instead of creating a new variable `functionsFlag` you can directly use the value of `cmdCustomFunctionsPath`.